### ISSUE TITLE
Read Paper from This list

### ISSUE THREAD

**Issue by chetandhembre** [chetandhembre/NN_Concepts#18]
https://github.com/terryum/awesome-deep-learning-papers#understanding--generalization--transfer

**Comment by chetandhembre** [comment]
- [x] - [Delving deep into rectifiers: Surpassing human-level performance on imagenet classification (2015), K. He et al.](https://github.com/chetandhembre/NN_Concepts/blob/master/papers/Delving%20Deep%20into%20Rectifiers-%20Surpassing%20Human-Level%20Performance%20on%20ImageNet%20Classification.md)
- [x] - [Dropout: A simple way to prevent neural networks from overfitting (2014), N. Srivastava et al.](https://github.com/chetandhembre/NN_Concepts/blob/master/papers/dropout.md)
